run_scans.py

```diff
@@ -98,7 +98,11 @@ def launchArgs(tool, shelf, slot, link, chamber, vfatmask, scanmin, scanmax, nev
     pass
   elif tool == "ultraLatency.py":
     scanType = "latency/trk"
-    dirPath = "%s/%s/%s/"%(dataPath,chamber_config[link],scanType)
+    dirPath = "%s%s/%s/"%(dataPath,chamber,scanType)
```

Comment:
Again, `chamber` is not defined here.

Additionally you're not formatted correctly, .e.g right now this will try to make a path:

```
dataPathchamber/scanType
```

Which doesn't exist; instead of

``` 
dataPath/chamber_config[link]/scanType
```

More thorough testing is needed.

Reply:
Regarding the formatting here, I did this because the variable ```dataPath``` is ```/data/bigdisk/GEM-Data-Taking/GE11_QC8/```, which already has a slash at the end. It did feel weird looking at it though, lol. In this case, should anything else be done? I'll also include an output of running the code while printing the values of the variables ```dataPath```, ```chamber```, and ```chamber_config[link]```, which I called ```chamber2```:
```
(py2.7)[bstone@gem904qc8daq src]$ run_scans.py -c eagle60 -g2 --L1Atime=250 --mspl=3 --pulseDelay=40 --scanmin=0 --scanmax=255 --internal --series --shelf=1 --tool=ultraLatency.py --voltageStepPulse
dataPath =  /data/bigdisk/GEM-Data-Taking/GE11_QC8/
chamber =  GE11-X-S-CERN-0003
chamber2 =  GE11-X-S-CERN-0003
dataPath =  /data/bigdisk/GEM-Data-Taking/GE11_QC8/
chamber =  GE11-X-S-CERN-0012
chamber2 =  GE11-X-S-CERN-0012
```

Reply:
But if a user sets up the `DATA_PATH` variable differently than how you do it, for example:

```
% echo $DATA_PATH 
/data/bigdisk/GEM-Data-Taking/GE11_QC8
```

There will be no trailing slash, and will thus result in an error.  However, having two `/` in a row will not.  Please revert this change.

Reply:
ah ok, gotcha